R/create_child_care_db.R

```diff
@@ -492,3 +492,129 @@ save_subset_child_care_db_03 <- function(pth, config, dev = TRUE) {
                                   the universe of child care data before subsetting.")
   }
 }
+
+#' @title Save a subset of the child care database for children 0-3 for a group of counties
+#' @param pth string. Path to the root directory to create the DB.
+#' @param config object. Object containing parameters to create the DB. 
+#' Config should be named list with 3 components: "label" for naming, "county_names" list of counties 
+#' to include, "tract_radius"
+#' @param dev boolean. St dev = FALSE to save the db to the production environment. Default is TRUE.
+#' @export
+save_subset_child_care_db_03_group <- function(pth, config, dev = TRUE) {
+  
+  if(file.exists(pth)) {
+    
+    load_env(file.path(pth))
+    
+    env <- new.env()
+    
+    env$county_codes <- tigris::list_counties(state = 48) %>% 
```

Comment:
Below is the config for the gulf coast workforce board. I had it using county codes before, but thought it would be easier/less room to make mistakes if I pulled the county codes from tigris as part of the function.

`  config <- list(label = "gulf_coast_wfb",
               county_names = c("Austin", 
                                "Brazoria",
                                "Chambers",
                                "Colorado",
                                "Fort Bend",
                                "Galveston",
                                "Liberty",
                                "Matagorda",
                                "Montgomery",
                                "Walker",
                                "Waller",
                                "Wharton"),
               tract_radius = 3)`